docs/glossary.md

```diff
@@ -554,12 +554,21 @@ react-intl
 WSGI
     The Web Server Gateway Interface (WSGI, pronounced _WIZ-ghee_) is a simple calling convention for web servers to forward requests to web applications or frameworks written in the Python programming language.
 
+ZODB Extension Objects
```

Comment:
it is "Zope Enterprise Objects" as far as I know. Did I wrote "Extension" here? Ops.

Reply:
I checked my commit, I did not ;)

Reply:
I copied it from https://github.com/plone/documentation/pull/1894/files#diff-79eb143dc64d651b159f035909f31d4bb50d8b8250fb8d9854142040fb14d206R40. I'll fix all instances.

Reply:
See PR https://github.com/plone/documentation/pull/1900

Reply:
ok, mea culpa!